pkg/scheduler/cache/cache.go

```diff
@@ -232,6 +232,33 @@ type defaultStatusUpdater struct {
 	vcclient   *vcclient.Clientset
 }
 
+func NewPodStatusClient(config *rest.Config) (*kubernetes.Clientset, error) {
+	configShallowCopy := *config
+	qps := os.Getenv("POD_STATUS_CLIENT_QPS")
+	if qps != "" {
+		qpsFloat, err := strconv.ParseFloat(qps, 32)
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse qps: %v", err)
+		}
+		configShallowCopy.QPS = float32(qpsFloat)
+	}
+
+	burst := os.Getenv("POD_STATUS_CLIENT_BURST")
+	if burst != "" {
+		burstInt, err := strconv.Atoi(burst)
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse burst: %v", err)
+		}
+		configShallowCopy.Burst = burstInt
+	}
+
+	client, err := kubernetes.NewForConfig(&configShallowCopy)
```

Comment:
Here, we can use `AddUserAgent` to specify an identity to distinguish from the default client.